db/seeds/db_populator.rb

```diff
@@ -130,7 +130,8 @@ def create_cases(casa_org, options)
         case_number: generate_case_number,
         transition_aged_youth: random_true_false
       )
-      CaseAssignment.create!(casa_case: new_casa_case, volunteer: volunteers.sample(random: rng))
+      casa_org_volunteers = volunteers.select { |volunteer| volunteer.casa_org_id = casa_org.id }
```

Comment:
I know this already got merged, but @patrickarnett this should be a `==` here, correct?

Reply:
Yes, I'll correct that.

Reply:
https://github.com/rubyforgood/casa/pull/1329